### PR TITLE
Add Scala - SBT example

### PR DESCRIPTION
**Pull request by daniel-shuy**

### CODE REVIEW

File: examples.md [hunk 1]
```diff
@@ -11,6 +11,7 @@
 - [Python - pip](#python---pip)
 - [Ruby - Gem](#ruby---gem)
 - [Rust - Cargo](#rust---cargo)
+- [Scala - SBT](#scala---sbt)
```
Comment (dhadka) [inline examples.md]: Please also add a link to this in README.md under Implementation Examples.

**Comment by joshmgross** [comment]
Awesome, thanks for this! This will resolve https://github.com/actions/cache/issues/59

**Comment by daniel-shuy** [comment]
Ah, just noticed https://github.com/actions/cache/issues/59#issuecomment-552846834. This example only works for SBT < 1.3 (with Ivy), or SBT 1.3+ with Coursier disabled (which will fallback to Ivy). Seems like the cache size limit is too small for Coursier at the moment.

**Comment by dhadka** [comment]
@daniel-shuy Will this example work with Coursier as-is once the cache size limit is increased?  We expect to rollout the size increase early next year.

**Comment by daniel-shuy** [comment]
> 
> 
> @daniel-shuy Will this example work with Coursier as-is once the cache size limit is increased? We expect to rollout the size increase early next year.

Unfortunately no, the configuration for Coursier is very different

**Pull request merged**